sdk/identity/identity/managed-identity-matrix.json

```diff
@@ -0,0 +1,19 @@
+{
+  "displayNames": {
+    "**/test-results.xml": ""
```

Comment:
Just curious why is this needed?

Reply:
If I recall it's for the pipeline test reports

Reply:
The values of each matrix entry become the title of the test job. This is here so we don't have names like `msi_image_deploymiresources_true_18_testresultsxml`